evaluators/text/ReciprocalRankEvaluator/tests/test_reciprocalrankevaluator.py

```diff
@@ -0,0 +1,18 @@
+import pytest
+
+from .. import ReciprocalRankEvaluator
+
+
+@pytest.mark.parametrize('actual, desired, score',
+[
+    ([],[], 0.0),
+    (['hey', 'hello', 'hi', 'heya'],[], 0.0),
```

Comment:
I think these tests are a little missleading. Normally uses `RankingEvaluations` to evaluate a list of `documentIds` or `scores`. I guess `hey, hello, hi...` can be `document identifiers` but if I come just to see the `tests` I get very confused.

Reply:
Okay, I will make changes for `documentIds`. I was considering doc content as the `actual` and `desired`